src/app/zap-templates/zcl/data-model/chip/target-navigator-cluster.xml

```diff
@@ -25,30 +25,30 @@ limitations under the License.
     <server init="false" tick="false">true</server>
     <!-- TODO: Add feature map once it is supported -->
     <description>This cluster provides an interface for UX navigation within a set of targets on a device or endpoint.</description>
-    <attribute side="server" code="0x0000" define="TARGET_NAVIGATOR_LIST" type="ARRAY" entryType="NavigateTargetTargetInfo" length="254" writable="false" optional="false">target navigator list</attribute>
+    <attribute side="server" code="0x0000" define="TARGET_NAVIGATOR_LIST" type="ARRAY" entryType="TargetInfo" length="254" writable="false" optional="false">target navigator list</attribute>
     <attribute side="server" code="0x0001" define="TARGET_NAVIGATOR_CURRENT_TARGET" type="INT8U" min="0x00" max="0xFF" writable="false" default="0x00" optional="true">current navigator target</attribute>
 
-    <command source="client" code="0x00" name="NavigateTarget" optional="false">
+    <command source="client" code="0x00" name="NavigateTargetRequest" response="NavigateTargetResponse" optional="false">
```

Comment:
Per spec the response here is "Launch" and there is no "NavigateTargetResponse" at all....  Please file spec issues as needed?

Reply:
Created the issue: https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/new